Omulosi/iReporter
# #162657975 Fix signup endpoint

#### What does this PR do?
Fixes the signup endpoint and ensures it is working correctly
#### Description of Task to be completed?
Refactor the signup endpoint to ensure that:
* A null/None value is not passed to the isadmin field when storing a user to the database
* email and password validation functions correctly validate emails and password respectively
* a valid user can be successfully stored in the database
#### How should this be manually tested?
clone this repository and check out the bg-fix-signup-endpoint-162657975 branch. Run the application as documented in the readme and check that a valid user can be successfully registered
#### Any background context you want to provide?
The changes are implemented as part of a feedback review
#### What are the relevant pivotal tracker stories?
#162657975
